src/rules/no-commented-out-tests.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { getDocsUrl } = require('./util');
+
+const message = 'Some tests seem to be commented';
+
+function hasTests(node) {
+  return /x?(test|it|describe)(\.\w+|\[['"]\w+['"]\])?\(.*?\)/.test(node.value);
```

Comment:
* Other than `xit`, there's also `fit`
* The closing parenthesis might cause trouble with multi-line tests in comments, we might need the `//s` regex modifier? Should add some multiline test cases

Reply:
Addressed both comments and added tests. The thing about newlines is that ESLint seems to create a separate comment for each line with line-level comments. We can't detect stuff like:

```js
// test
// (
// 'x
// )
```

..which, I think, is an OK limitation?

Reply:
For block comments as well though?

Reply:
Actually, best thing might be to just not look for the closing parenthesis? `// test(` already looks suspicious enough

Reply:
Yep, I did just that in the last commit. Let me add a multiline comment test as well.